charts/cp-zookeeper/templates/tests/test-zk-connection.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
```

Comment:
@sflandergan ill helm remove this pod after test?

Reply:
@gAmUssA unfortunately not. Seems this will only be supported with helm 3.x: https://github.com/helm/helm/issues/3211